src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -84,22 +112,7 @@ public boolean doMonitor(ServiceEmitter emitter)
       emitter.emit(builder.build("jvm/pool/init",      usage.getInit()));
     }
 
-    // jvm/gc
-    for (GarbageCollectorMXBean gc : ManagementFactory.getGarbageCollectorMXBeans()) {
-      final Map<String, Long> diff = gcDiff.to(gc.getName(), ImmutableMap.of(
-          "jvm/gc/time",  gc.getCollectionTime(),
-          "jvm/gc/count", gc.getCollectionCount()
-      ));
-      if (diff != null) {
-        final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder()
-            .setDimension("gcName", gc.getName());
```

Comment:
Would be cool to have GC name as "gcName" dimension in the new implementation of server metrics.